File: U1/EXAMEN/BanquitoServer_Soap_DotNet_GR01/BanquitoServer_Soap_DotNet_GR01/DTOs/MontoMaximoCreditoDTO.cs
using System.Runtime.Serialization;

namespace BanquitoServer_Soap_DotNet_GR01.DTOs
{
    /// <summary>
    /// DTO para respuesta de monto máximo de crédito
    /// </summary>
    [DataContract]
    public class MontoMaximoCreditoDTO
    {
        [DataMember]
        public string Cedula { get; set; }

        [DataMember]
        public decimal MontoMaximo { get; set; }

        [DataMember]
        public decimal PromedioDepositos { get; set; }

        [DataMember]
        public decimal PromedioRetiros { get; set; }

        [DataMember]
        public string Mensaje { get; set; }
    }
}

File: U2/PAO202551_ELECTRO_BDDNUBE_RESTFULL_JAVA_GR01/04. SERVIDOR/BanquitoServer_Restfull_Java_GR01/src/main/java/ec/edu/monster/DTOs/MontoMaximoCreditoDTO.java
package ec.edu.monster.DTOs;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.math.BigDecimal;

/**
 * DTO para respuesta de monto máximo de crédito
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class MontoMaximoCreditoDTO {
    private String cedula;
    private BigDecimal montoMaximo;
    private BigDecimal promedioDepositos;
    private BigDecimal promedioRetiros;
    private String mensaje;
}
